svc-api/handle/handlers_test.go

```diff
@@ -490,3 +490,34 @@ func TestManagersMethodNotAllowed(t *testing.T) {
 	e.PATCH("/redfish/v1/Managers/{id}").Expect().Status(http.StatusMethodNotAllowed)
 	e.DELETE("/redfish/v1/Managers/{id}").Expect().Status(http.StatusMethodNotAllowed)
 }
+
```

Comment:
why are we adding test cases now and not before development?

Reply:
this test case was missed in adding test cases PR